app/views/public_pages/template_index.html.erb

```diff
@@ -16,11 +16,11 @@
       <div class="table-responsive">
       <table class="table table-hover tablesorter">
         <thead>
-          <% if @templates.count > TABLE_FILTER_MIN_ROWS %>
+          <% if @plans.count > TABLE_FILTER_MIN_ROWS %>
```

Comment:
Think this is a copy-paste error. Update to `@templates.count` and  `colspan="3"` and `public_templates_path, placeholder: _('Filter templates')}` below